charts/zitadel/Chart.yaml

```diff
@@ -2,11 +2,11 @@ apiVersion: v2
 name: zitadel
 description: A Helm chart for ZITADEL
 type: application
-appVersion: "v2.41.0"
-version: 7.2.0
+appVersion: "v2.41.1"
+version: 7.3.0
 kubeVersion: ">= 1.21.0-0"
 icon: https://zitadel.com/zitadel-logo-dark.svg
 maintainers:
   - name: zitadel
     email: support@zitadel.com
-    url: https://zitadel.com
+    url: https://zitadel.com
```

Comment:
```suggestion
    url: https://zitadel.com

```

Reply:
Without the newline, linting fails

Reply:
The new line is added back now. Sorry about that.